### PR TITLE
Fix crash if numpy doesn't have version

### PR DESCRIPTION
**Pull request by tushar-deepsource**
## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✓   | :bug: Bug fix          |

## Related Issue

Resolves https://github.com/PyCQA/pylint/issues/7868

### CODE REVIEW

**Comment by mbyrnepr2** [comment]
Thanks for the MR @tushar-deepsource.
I'm hesitant about the fix because, with a local `numpy.py` now getting imported, we run into `E1101: Module 'numpy' has no 'ndarray' member (no-member)` (using the [code sample](https://github.com/PyCQA/pylint/issues/7868#issue-1468175279) linked from the issue). Perhaps an alternative is to identify if there is a local `numpy.py` & try to ignore that one?

**Review by mbyrnepr2 — APPROVED** [review]

**Comment by tushar-deepsource** [comment]
Is this waiting on something?

**Comment by mbyrnepr2** [comment]
Hey @tushar-deepsource I think we can merge. I just realized there is no Change log entry. Can you add that, then I think we should be good to go.

**Review by Pierre-Sassoulas — APPROVED** [review]

**Review by DanielNoord — APPROVED** [review]

**Comment by DanielNoord** [comment]
@tushar-deepsource Could you force-push the branch? Jobs seem to broken..

File: ChangeLog [hunk 1]
```diff
@@ -24,6 +24,9 @@ Release date: TBA
 
   Closes PyCQA/pylint#5225
 
+* Fix crash if `numpy` module doesn't have `version` attribute
```
Comment (mbyrnepr2) [inline ChangeLog:27]:
```suggestion
* Fix crash if ``numpy`` module doesn't have ``version`` attribute. 
```

Comment (mbyrnepr2) [reply ChangeLog:27]: Just to be consistent with other logs

**Pull request merged**